packages/auth/package.json

```diff
@@ -13,17 +13,17 @@
   "devDependencies": {
     "@session/eslint-config": "workspace:*",
     "@session/typescript-config": "workspace:*",
-    "@types/react": "^18.3.1",
-    "@types/react-dom": "^18.3.0",
+    "@types/react": "npm:types-react@19.0.0-rc.1",
+    "@types/react-dom": "npm:types-react-dom@19.0.0-rc.1",
```

Comment:
Do you still need `react-dom` typings if you don't have `react-dom` installed?

Reply:
I guess you are making sure nothing goes weird with peer deps or something by being explicit.